### ISSUE TITLE
The Debian and Arch packages do not ship with `libsoljson`

### ISSUE THREAD

**Issue by cag** [ethereum/solidity#2255]
This means no FFI here: https://github.com/ethereum/py-solc/issues/9

**Comment by axic** [comment]
This is not part of the build system at the moment. Is it possible with Python FFI to load a shared library as is?

**Comment by cag** [comment]
Yes, but this practically means shipping either the whole solidity source as a submodule or incorporating it into dependent project build processes to make platform wheels available on pip for example. In any case, just having it exposed to CMake should be enough if I'm not mistaken. I suppose there may be some documentation work that might need to be done before this is shippable as part of the official release depending on Solidity policies on the matter, but ultimately, I feel like addressing it here is the right way to go.

**Comment by chriseth** [comment]
I fear providing such packages is currently out of scope.

**Comment by axic** [comment]
Depends on #2864.

**Comment by chriseth** [comment]
What is to be done here now that the dependency is closed?

**Comment by cag** [comment]
I feel this issue is stale as it holds some outdated assumption, but the issue downstream may be addressed directly now. I've edited the link in the original post to point to the right location. Closing this now.
